include/enemy.h

```diff
@@ -12,6 +12,7 @@ class Enemy : public Moveable {
         float velocityY;
         float xpos;
         float ypos;
+        int colorType;
```

Comment:
I have a variable just like this in my branch but it's just called type. So I would rename this to "type" to avoid merge conflicts.